FiveCalls/FiveCalls/IssuesContainerViewController.swift

```diff
@@ -38,15 +39,40 @@ class IssuesContainerViewController : UIViewController, EditLocationViewControll
         
         issuesVC.didMove(toParentViewController: self)
         issuesViewController = issuesVC
+        
+        let effectView = UIVisualEffectView(frame: headerContainer.bounds)
+        effectView.translatesAutoresizingMaskIntoConstraints = false
+        effectView.effect = UIBlurEffect(style: .light)
```

Comment:
I'm a Swift newbie so I had to try a few things before it worked... Needed to add () after the closing curly brace.